app/javascript/schools/courses/components/students_editor/StudentsEditor__BulkImportForm.res

```diff
@@ -66,6 +69,7 @@ let reducer = (state, action) =>
   switch action {
   | UpdateFileInvalid(fileInvalid) => {...state, fileInvalid: fileInvalid}
   | ClearCSVData => {...state, fileInfo: None, fileInvalid: None, csvData: []}
+  | ChangeNotifyStudents => {...state, notifyStudents: !state.notifyStudents}
```

Comment:
```suggestion
  | ToggleeNotifyStudents => {...state, notifyStudents: !state.notifyStudents}
```

Reply:
There's a typo in the suggestion: `ToggleeNotifyStudents` 😅